.travis.yml

```diff
@@ -5,6 +5,15 @@ language: node_js
 cache:
   directories:
   - node_modules
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
it didn't seem to work without the check but i'l test again https://github.com/babel/babel/pull/4556